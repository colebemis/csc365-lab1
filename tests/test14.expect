Enter query: Unrecognized command. Possible commands:
- S[tudent]: <lastname> [B[us]]
- T[eacher]: <lastname>
- B[us]: <number>
- G[rade]: <number> [H[igh]|L[ow]]
- A[verage]: <number>
- I[nfo]
- Q[uit]
Enter query: 