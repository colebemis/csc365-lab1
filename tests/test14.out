Enter query: Unrecognized command. Possible commands:
- S[tudent]: <lastname> [B[us]]
- T[eacher]: <lastname>
- B[us]: <number>
- G[rade]: <number> [H[igh]|L[ow]|T[eacher]]
- C[lassroom]: <number> [T[eacher]]
- A[verage]: <number>
- I[nfo]
- E[nrollment]
- R[aw]: [grade=<number>] [bus=<number>] [teacher=<lastname>]
- Q[uit]
Enter query: 